charts/shoot-core/charts/calico/templates/calico.yaml

```diff
@@ -8,6 +8,16 @@ metadata:
 
 ---
 
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: calico-typha
```

Comment:
Nice catch, it's actually invalid in the official calico manifest itself: https://docs.projectcalico.org/v3.3/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml
Let's fix that on our side.

Reply:
We should use a different service account, because those components require different PSPs.

I'll update our calico-typha deployment to use that `ServiceAccount`